ros_buildfarm/templates/doc/doc_independent_site_job.xml.em

```diff
@@ -0,0 +1,135 @@
+<project>
+  <actions/>
+  <description>Generated at @ESCAPE(now_str) from template '@ESCAPE(template_name)'</description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+@(SNIPPET(
+    'property_log-rotator',
+    days_to_keep=365,
+    num_to_keep=100,
+))@
+@[if job_priority is not None]@
+@(SNIPPET(
+    'property_job-priority',
+    priority=job_priority,
+))@
+@[end if]@
+@(SNIPPET(
+    'property_requeue-job',
+))@
+  </properties>
+@(SNIPPET(
+  'scm_git',
+   url=upload_repository_url,
+   branch_name=upload_repository_branch or 'gh-pages',
+   git_ssh_credential_id=upload_credential_id,
+   relative_target_dir='site',
+   refspec=None,
+))@
+  <assignedNode>@(node_label)</assignedNode>
+  <canRoam>false</canRoam>
+  <disabled>false</disabled>
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <triggers>
+@(SNIPPET(
+    'trigger_timer',
+    spec='H 3 * * *',
```

Comment:
How long does the test job take and how often do we want it to run?

Reply:
I presume it'll take somewhere between one and two hours on a good network connection. And I understand we want this to be built on a daily basis (just as it is for other independent doc types). We did not discuss any other form of triggering.

Reply:
I had to remind myself of the Jenkins cron spec behavior. This runs once a day at an idle time during 3-4AM. :+1: